src/main/webapp/display-event.html

```diff
@@ -48,6 +48,15 @@
             <span class="tag environment">environment</span>
           </div>
         </div>
+        <div class="footer">
+          <p>
+            Icons made by 
+            <a href="https://www.flaticon.com/authors/freepik"
+            title="Freepik"> Freepik</a> from 
+            <a href="https://www.flaticon.com/" title="Flaticon">
+            www.flaticon.com</a>
```

Comment:
Should these lines have 4 spaces indentation since they're continuation?

Reply:
Yes, thanks!